test/e2e/instrumentation/monitoring/prometheus.go

```diff
@@ -120,7 +120,7 @@ func exportCustomMetricFromPod(f *framework.Framework, consumerName string, metr
 func exportCustomMetricFromService(f *framework.Framework, consumerName string, metricValue int) *common.ResourceConsumer {
 	serviceAnnotations := map[string]string{
 		"prometheus.io/scrape": "true",
-		"prometheus.io/path":   "/Metrics",
+		"prometheus.io/path":   "/metrics",
```

Comment:
I think only this e2e test.

Reply:
that seems useful to preserve. would it be clearer if this image used an obviously non-standard endpoint like "/custom-metrics-endpoint"?

Reply:
if I understand the test correctly, part of what this is testing is that the `prometheus.io/path` is respected for scraping metrics from non-standard paths. if that is a feature that we support (and it appears we do), we must have test coverage for it to have confidence it continues working.

/cc @brancz

Reply:
This is highly specific to how Prometheus is configured, as in Prometheus itself has no knowledge of the `prometheus.io/path` annotation. I personally don't think this is something we should be testing at all.

Reply:
maybe I'm misunderstanding this, then... what does the `prometheus.io/path` annotation do, why is this test using the non-standard path, and how is it working currently?

Reply:
The `prometheus.io/path` annotation is simply (in this specific configuration that is being used) a Prometheus relabeling rule that configures the path, my point is that the annotation itself is free form, it could just as well be `mycompany.io/prometheus-metrics-path`. This is just a matter of how Prometheus is configured. The point I'm trying to make is that I'm not sure testing Prometheus' configuration/discovery system here is something we should be doing.

Reply:
I thought that here we were testing if the resource consumer was exposing the fake Prometheus-format metrics correctly, not the Prometheus configuration or anything else.

Reply:
if that's what we want to test then as is, is ok or just skip the annotation entirely as `/metrics` is the default

Reply:
Usage of "/Metrics" endpoint in resource-consumer precedes implementation of Prometheus tests and adding this addon. Prometheus tests just adapted to what was already available to skip breaking dependencies and releasing new images.  

Resource consumer is not owned by sig-instrumentation, by it's original usage for HPA tests I it was created by sig-autoscaling.

Reply:
if the test owner doesn't think the distinction is important, I won't object... just wanted to raise the question to make sure they were aware of the previously covered case being dropped, in case this was the only coverage of it

Reply:
@liggitt Who is the test owner? can we ping him? What is blocking this PR?